torch/nn/modules/activation.py

```diff
@@ -831,7 +831,7 @@ def extra_repr(self) -> str:
 class MultiheadAttention(Module):
     r"""Allows the model to jointly attend to information
     from different representation subspaces.
-    See reference: Attention Is All You Need
+    See reference: Attention Is All You Need(https://arxiv.org/abs/1706.03762)
```

Comment:
See \`Attention Is All You Need \<https://arxiv.org/abs/1706.03762>`_

Reply:
Thank you for reviewing my PR. I updated the reference in the docstring.